src/main/java/frc/robot/Robot.java

```diff
@@ -279,10 +279,15 @@ public void robotInit() {
 
     controller
         .y()
-        .and(() -> climber.getRotations() > 0.9 * ClimberSubsystem.CLIMBER_MAX_ROTATIONS)
+        .and(() -> climber.getRotations() > 0.9 * ClimberSubsystem.CLIMB_ROTATIONS)
         .onTrue(
             Commands.sequence(
-                    climber.retractClimbCmd().until(() -> climber.getRotations() < 0.05), // TODO check
+                    climber
+                        .retractClimbCmd()
+                        .until(
+                            () ->
+                                climber.getRotations()
+                                    < 1.1 * ClimberSubsystem.CLIMBER_MIN_ROTATIONS),
                     Commands.waitUntil(() -> controller.y().getAsBoolean()),
```

Comment:
this can be simplified to `controller.y()`

Reply:
waitUntil needs a boolean supplier and controller.y() is a trigger